modules/ocf_kubernetes/files/ingress_vhost.yaml

```diff
@@ -0,0 +1,13 @@
+# Add Ingress Rules to route by Host Header
```

Comment:
Forgot about this. This file should not be here... These should be in a separate repo with our kubernetes manifests. That said, I'll leave it here for now since it's a good reference to see what the `ingress-nginx` controller does.